src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationDefaults.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+
+namespace AspNet.Security.OAuth.Alipay
+{
+    /// <summary>
+    /// Default values for Alipay authentication.
+    /// </summary>
+    public static class AlipayAuthenticationDefaults
+    {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationScheme.Name"/>.
+        /// </summary>
+        public const string AuthenticationScheme = "Alipay";
+
+        /// <summary>
+        /// Default value for <see cref="AuthenticationScheme.DisplayName"/>.
+        /// </summary>
+        public const string DisplayName = "Alipay";
+
+        /// <summary>
+        /// Default value for <see cref="AuthenticationSchemeOptions.ClaimsIssuer"/>.
+        /// </summary>
+        public const string Issuer = "Alipay";
+
+        /// <summary>
+        /// Default value for <see cref="RemoteAuthenticationOptions.CallbackPath"/>.
+        /// </summary>
+        public const string CallbackPath = "/signin-alipay";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.AuthorizationEndpoint"/>.
+        /// </summary>
+        public const string AuthorizationEndpoint = "https://openauth.alipay.com/oauth2/publicAppAuthorize.htm";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.TokenEndpoint"/>.
+        /// </summary>
+        public const string TokenEndpoint = "https://openapi.alipay.com/gateway.do";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.UserInformationEndpoint"/>.
+        /// </summary>
+        public const string UserInformationEndpoint = "https://openapi.alipay.com/gateway.do";
```

Comment:
Is it correct that the same URL is used for tokens and users? Are the responses identical, or is there some other parameter used to differentiate between getting a token and getting a users' details?

Reply:
Yes.
Get token : https://openapi.alipay.com/gateway.do?method=alipay.system.oauth.token... 
Get user info : https://openapi.alipay.com/gateway.do?method=alipay.user.info.share...